core_contracts/governance/governance.py

```diff
@@ -979,4 +980,11 @@ def setRouter(self, _router: Address):
         """
         Introduces the transaction router SCORE
         """
-        self.addresses._router.set(_router)
\ No newline at end of file
+        self.addresses._router.set(_router)
+
+    def BIP7_fixes_update(self):
+        proposal = ProposalDB(var_key=8, db=self.db)
+        proposal.status.set('Succeeded')
+
+        _action = '{"setRebalancingThreshold":{"_value":30000000000000000}}'
+        proposal.actions.set(_action)
```

Comment:
Looks like the evaluateVote method also set active to False, so we will have to set it back to True in order to call evaluateVote again.